example/main.tf

```diff
@@ -1,19 +1,44 @@
+locals {
+  permissions_boundary_name = var.permissions_boundary == null ? null : reverse(split("/", var.permissions_boundary))[0]
+}
+
 provider "aws" {
   region = var.region
 }
 
 module "thin_egress_app" {
-  source = "https://s3.amazonaws.com/asf.public.code/thin-egress-app/tea-terraform-build.16.zip"
+  # source = "https://s3.amazonaws.com/asf.public.code/thin-egress-app/tea-terraform-build.16.zip"
+  source = "../../thin-egress-app/terraform"
 
   auth_base_url              = "https://uat.urs.earthdata.nasa.gov"
   bucket_map_file            = var.tea_bucket_map_file
   bucketname_prefix          = ""
   config_bucket              = var.tea_config_bucket
   domain_name                = var.tea_domain_name
-  permissions_boundary_name  = var.permissions_boundary
+  lambda_code_s3_key         = "thin-egress-app/tea-code-build.16.zip"
+  permissions_boundary_name  = local.permissions_boundary_name
   private_vpc                = var.vpc_id
   stack_name                 = var.tea_stack_name
   stage_name                 = var.tea_stage_name
+  template_body              = file("${path.module}/../../thin-egress-app/cloudformation/thin-egress-app.yaml")
   vpc_subnet_ids             = var.tea_subnet_ids
   urs_auth_creds_secret_name = var.tea_urs_auth_creds_secret_name
 }
+
+module "s3_credentials_endpoint" {
+  source = "../packages/s3-credentials-endpoint"
```

Comment:
this should be some kind of environment/config variable. The way the package is currently written, if you’re using npm and this package is listed as a dependency, this should be something like:
```
source = “node_modules/@cumulus/s3-credentials-endpoint”
```